### PR TITLE
Feature/project deploy

### PR DESCRIPTION
**Pull request by Felipe-Perez-Ferraro**
# Home page

- When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
- Along with the category name, you will display some numeric values per category that come from the API.
- When the user clicks (or taps) on a category item, the application navigates to the details page.
Details page

- In the details page, the webapp retrieves data from the API to show detailed data in the category.
- When the user clicks on the "Back" button (<), the user navigates to the home page.

# Testing requirements

- Create unit tests for pure functions in your Redux code.
- Create integration tests for your application using the React Testing Library.
- You may need to mock the access to the API, so that your tests don't send actual requests.
- You may need to mock the connection to the Redux Store.

### CODE REVIEW

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @Felipe-Perez-Ferraro 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️ 
✔️ Used Gitflow 
✔️ Work documented professionally

## Suggested changes

- [ ] Good job creating your React capstone project 👏 👏 . However you should provide a `video presentation` containing the follow details 🥇 

          1. The maximum length of the video should be  5 minutes.
          2. The video should contain a clear and objective description of the project in 
          professional English. The topic and purpose of the project are made clear.
          3. The video should contain a demo of the project, highlighting all the project features.
          4. The video should contain a highlight about a piece of the code the student finds 
          interesting or is proud of making.

- [ ] As you have used someone else design, it is important to give a credit to him/her as a privilege. This helps anyone who visits your website but is interested in the design you have followed get to know the source 💡. Moreover, it is among the project requirements to give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) you used. Please add her here in the Acknowledgement section and the link to the design. 😃 👍

Capture


- [ ] A live demo link should be present in your README file , however you don't have it , Kindly render your project using [Netlify](https://www.netlify.com/) to meet the requirement 👍 

Capture

- [ ] Great job with the technical part of your  project, I admire your effort so far, You've done well. I just wanted to point out something concerning the search. As it's a requirement to add a search functionality, so a user could filter  data from the API , I don't seem to see it working as intended. When a user searches or want to filter a company, it should show up what matches the search e.g, the name of the company, but on the contrary, it doesn't you did not add that feature. Please, try to add this implementation to meet this part of the project requirement. :+1: 

### They use search to filter 👍 

Capture


- [ ] Good job adding tests 👍 however it could be better if you add text for the components you have created in your directory 😄 . You only have two tests 👍 and zero snapshots 👍 . Kindly complete that 💯 

Capture


_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -4,13 +4,15 @@ import {
...
 import { Link } from 'react-router-dom';
 import { screenStyle } from '../tailwindStyles';
 
 const goBack = <FontAwesomeIcon icon={faAngleLeft} />;
 const user = <FontAwesomeIcon icon={faMicrophone} />;
 const settings = <FontAwesomeIcon icon={faGear} />;
+const search = <FontAwesomeIcon icon={faMagnifyingGlass} />;
 
 function Navbar() {
   return (
```
Comment (thecodechaser) [inline src/components/Navbar.jsx]: - [ ] As requested by the previous reviewer, Please make sure to add search functionalities where users can filter items by some parameters(Company name in your case). You just added the search icon but please also add the search functionalities as well.

File: src/__tests__/companies.test.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ describe('companiesSlice tests', () => {
     fetchApi.mockResolvedValue(mockData);
     await store.dispatch(fetchCompaniesThunk('symbol'));
 
-    render(
+    const { yearInfo } = render(
       <Provider store={store}>
         <MemoryRouter initialEntries={['/AAPL/2022']}>
           <Routes>
```
Comment (thecodechaser) [inline src/__tests__/companies.test.js]: - [ ] As requested by the previous reviewer, Please add tests for all components in the `/components` directory.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Felipe-Perez-Ferraro👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- Good job with React functionalities. ✔️
- All tests are passed. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — APPROVED** [review]
Hi @Felipe-Perez-Ferraro, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job completing your capstone 👍 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**